src/explore-education-statistics-admin/src/pages/release/data/components/ReleaseDataUploadsSection.tsx

```diff
@@ -420,13 +420,12 @@ const ReleaseDataUploadsSection = ({
           onCancel={() => setCancelDataFile(undefined)}
           onConfirm={async () => {
             try {
+              setCancelDataFile(undefined);
```

Comment:
Thinking it'd be a good idea to either:

- Move this to after the try/catch so that we always close the modal
- Show some error message if the cancellation fails

I'm kinda inclined to think we should at least show a message if the cancellation request failed for whatever reason.

We can leave the `setFileCancelling` up here though :slightly_smiling_face: